src/Instrumentation/Symfony/src/SymfonyInstrumentation.php

```diff
@@ -113,7 +113,12 @@ public static function register(): void
                     return;
                 }
 
-                if ($response->getStatusCode() >= Response::HTTP_BAD_REQUEST) {
+                // Do not set error status on server spans for 4xx responses
+                if (!$span->getKind(SpanKind::KIND_SERVER) && $response->getStatusCode() >= Response::HTTP_BAD_REQUEST) {
```

Comment:
This only seems possible if this is a subrequest (internal redirect) in which case span kind is `INTERNAL`. I assume it was an intentional choice to use error status specifically for 400 codes from internal redirects, as the specification does not cover this scenario at all. Since you already have a comment here, perhaps it could mention that this case is for subrequests/internal spans.

Reply:
Good point. This case can only happen with span kind Internal. The intention was to have this case mainly for client span kinds. Which can't happen here. 
I will remove this case, to ONLY set error status on >= 500 status codes. This is the same behaviour as implemented in Laravel.

Reply:
looks ok to me, can it be tested by adding/modifying our existing tests?